tools/custom-organization-policy-library/samples/constraints/gke/gkeRequireCOSImage.yaml

```diff
@@ -1,8 +1,7 @@
 name: organizations/11111111/customConstraints/custom.gkeRequireCOSImage
 resource_types:
-- container.googleapis.com/Cluster
 - container.googleapis.com/NodePool
-condition: 1 == 0
+condition: resource.config.imageType != "COS_CONTAINERD"
```

Comment:
Same here any docs ?

Reply:
This condition is similar to GKE custom organizational policies based on CIS Benchmarks in Github. No documentation was found as of now

Reply:
have u tested it ? with a case using containerd and a case using ubuntu.
Do you confirm it works ?

Reply:
![Screenshot 2024-04-16 10 22 53 AM](https://github.com/vannicktrinquier/professional-services/assets/162395680/e993df92-c964-4aaa-8608-1302a161cc89)
Yes Vannick, the constraint works when selecting Ubuntu. Will confirm that it is consistent throughout.

Reply:
cool